api/client/canary/aws.pb.go

```diff
@@ -113,13 +113,16 @@ type AwsAccount struct {
 	// The profile name to use when resolving AWS credentials. Typically found in
 	// `~/.aws/credentials`. Defaults to `default`.
 	ProfileName string `protobuf:"bytes,5,opt,name=profileName,proto3" json:"profileName,omitempty"`
-	// The endpoint used to reach the service implementing the AWS API. Typical
-	// use is with Minio.
+	// The endpoint used to reach the service implementing the S3 API. Typically
+	// you would use this with an S3 clone, like Minio.
 	Endpoint string `protobuf:"bytes,6,opt,name=endpoint,proto3" json:"endpoint,omitempty"`
 	// The default access key used to communicate with AWS.
 	AccessKeyId string `protobuf:"bytes,7,opt,name=accessKeyId,proto3" json:"accessKeyId,omitempty"`
-	// If enabling S3, include CONFIGURATION_STORE and/or OBJECT_STORE in this
-	// list.
+	// Use METRICS_STORE for AWS monitoring.
```

Comment:
Same comment as @maggieneterval 
AWS doesn't have `METRICS_STORE` as an option.

This would likely come as its own CloudWatch integration.
See: https://github.com/spinnaker/kayenta/issues/757